sceptre/config/reader.py

```diff
@@ -342,12 +346,10 @@ def read(self, rel_path, base_config=None):
         # Adding properties from class
         config = {
             "project_path": self.context.project_path,
-            "stack_group_path": directory_path,
```

Comment:
As far as I could tell, the only place this `stack_group_path` was being used was in the unit tests. So I've removed it.